actors/common/src/main/java/org/sunbird/feed/FeedUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.sunbird.feed;
+
+import java.util.*;
+import java.util.stream.IntStream;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.bean.ShadowUser;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.feed.impl.FeedFactory;
+import org.sunbird.models.user.Feed;
+import org.sunbird.models.user.FeedAction;
+import org.sunbird.models.user.FeedStatus;
+
+/** this class will be used as a Util for inserting Feed in table */
+public class FeedUtil {
+  private static IFeedService feedService = FeedFactory.getInstance();
+
+  public static void saveFeed(ShadowUser shadowUser, List<String> userIds) {
+    Map<String, Object> reqMap = new HashMap<>();
+    reqMap.put(JsonKey.USER_ID, userIds.get(0));
+    reqMap.put(JsonKey.CATEGORY, FeedAction.ORG_MIGRATION_ACTION.getfeedAction());
+    List<Feed> feedList = feedService.getRecordsByProperties(reqMap);
+    int index = getIndexOfMatchingFeed(feedList);
+    if (index == -1) {
+      feedService.insert(createFeedObj(shadowUser, userIds.get(0)));
+    } else {
+      Map<String, Object> data = feedList.get(index).getData();
+      List<String> channelList = (List<String>) data.get(JsonKey.PROSPECT_CHANNELS);
+      channelList.add(shadowUser.getChannel());
+      feedService.update(feedList.get(index));
+    }
+  }
+
+  private static Feed createFeedObj(ShadowUser shadowUser, String userId) {
+    Feed feed = new Feed();
+    feed.setPriority(1);
+    feed.setCreatedBy(shadowUser.getAddedBy());
+    feed.setStatus(FeedStatus.UNREAD.getfeedStatus());
+    feed.setCategory(FeedAction.ORG_MIGRATION_ACTION.getfeedAction());
+    Map<String, Object> prospectsChannel = new HashMap<>();
```

Comment:
1. During object creation time we can pass priority, status and category  (no need here to set again at least as of now)
2. prospectsChannel is storing only one object , so better to put capacity as double only in this case 2 , to avoid extra object creation
same can be done for list as well.